include/exadg/incompressible_navier_stokes/spatial_discretization/generalized_newtonian_model.h

```diff
@@ -0,0 +1,164 @@
+/*  ______________________________________________________________________
+ *
+ *  ExaDG - High-Order Discontinuous Galerkin for the Exa-Scale
+ *
+ *  Copyright (C) 2023 by the ExaDG authors
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *  ______________________________________________________________________
+ */
+
+#ifndef INCLUDE_EXADG_INCOMPRESSIBLE_NAVIER_STOKES_SPATIAL_DISCRETIZATION_GENERALIZED_NEWTONIAN_MODEL_H_
+#define INCLUDE_EXADG_INCOMPRESSIBLE_NAVIER_STOKES_SPATIAL_DISCRETIZATION_GENERALIZED_NEWTONIAN_MODEL_H_
+
+// ExaDG
+#include <exadg/incompressible_navier_stokes/spatial_discretization/viscosity_model_base.h>
+
+namespace ExaDG
+{
+namespace IncNS
+{
+/*
+ *  Generalized Newtonian model.
+ */
+template<int dim, typename Number>
+class GeneralizedNewtonianModel : public ViscosityModelBase<dim, Number>
+{
+private:
+  typedef GeneralizedNewtonianModel<dim, Number> This;
+  typedef ViscosityModelBase<dim, Number>        Base;
+
+  typedef dealii::LinearAlgebra::distributed::Vector<Number> VectorType;
+
+  typedef dealii::VectorizedArray<Number>                         scalar;
+  typedef dealii::Tensor<2, dim, dealii::VectorizedArray<Number>> tensor;
+
+  typedef std::pair<unsigned int, unsigned int> Range;
+
+  typedef CellIntegrator<dim, dim, Number> CellIntegratorU;
+  typedef FaceIntegrator<dim, dim, Number> FaceIntegratorU;
+
+public:
+  /*
+   *  Constructor.
+   */
```

Comment:
We can actually start to use Doxygen documentation with ( `/**` ) instead of block comments, now that we also have it in our contributing guidelines 😆. And we should do it for all members if possible.

Reply:
Yes, here for this PR it is enough to add a second * in the first row of the comment. This can then be understood by doxygen.